### PR TITLE
Hotfix profile page icons and styling

### PR DESCRIPTION
**Pull request by bejow**
I cleaned up the Profile Page and adjusted the styling therefor I added sass styling so that we can work with css variables (breakpoints, colors, mixins, etc). This will help when we start to make our drive more responsiv.

### CODE REVIEW

File: src/functional_components/Navigation/Navigation.module.css [hunk 1]
```diff
@@ -53,7 +53,7 @@
     align-items: center;
     justify-content: center;
     height: 100%;
-    padding: 0 20px;
+    padding: 0 20px 0 0;
```
Comment (ludwigschubi) [inline src/functional_components/Navigation/Navigation.module.css]: padding-left would be cleaner

File: src/functional_components/ProfilePage/ProfilePage.js [hunk 2]
```diff
@@ -83,30 +76,33 @@ export const ProfilePage = ({
...
-                                setValue={(value) =>
-                                    updateUserData('name', value)
-                                }
-                                className={styles.nameLabel}
-                                placholder="Enter Name.."
-                            />
-                            <div className={styles.webIdLabel}>
-                                {user.webId.replace('/profile/card#me', '')}
+                        <div className={styles.headDataContainer}>
+                            <div>
```
Comment (ludwigschubi) [inline src/functional_components/ProfilePage/ProfilePage.js]: this div is useless

File: src/functional_components/ProfilePage/ProfilePage.module.scss [hunk 3]
```diff
@@ -195,7 +176,9 @@ svg:hover {
 }
 
 .editWrapper {
-    grid-area: editIcon;
+    position: absolute;
+    top: 0;
+    right: -32px;
```
Comment (ludwigschubi) [inline src/functional_components/ProfilePage/ProfilePage.module.scss]: 2em is cleaner since we use 4px grid/spacing

File: styles/constants.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// breakpoints, color values, font sizes, margins
+
+//src https://www.mediaevent.de/css/breakpoints.html
+//breakpoints min width
+$screen_xs: 0px;
```
Comment (ludwigschubi) [inline styles/constants.scss]: maybe use the iphone 5s screen width here

File: styles/constants.scss [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+$screen_s: 567px;
+$screen_m: 768px;
+$screen_l: 992px;
+$screen_xl: 1200px;
+
+$light_grey: #f4f4f4;
+$dark_grey: #e2e2e6;
+$light_blue: #6db7db;
+$danger_red: #bd0000;
```
Comment (ludwigschubi) [inline styles/constants.scss]: 👍

File: styles/constants.scss (hunk 4 above)
Comment (bejow) [reply styles/constants.scss]: But what if there are smaller screens?

File: src/functional_components/ProfilePage/ProfilePage.module.scss (hunk 3 above)
Comment (bejow) [reply src/functional_components/ProfilePage/ProfilePage.module.scss]: Mhhh after using 2em and testing it with smaller screens I'm not that satisfied with 2em. Anyway I think we could come up with a new Layout for ProfilePage.

**Comment by bejow** [comment]
@ludwigschubi I merged the master, implemented your feedback and tests are now passing, could you review one more time please?

**Pull request merged**